src/main/webapp/js/captions.js

```diff
@@ -541,6 +550,30 @@ function setBookmarkButton() {
     });
 }
 
+function setCaptionsButton() {
+    // append full-captions button
+    $("#timestamp-timeline").append('<button id="fullcap-button"><i style="font-size:24px" class="fa fa-cc"></i></button>');
+    document.getElementById("FullCap").innerHTML = "";
+
+    // Display captions on click
+    $('#fullcap-button').click(function() {
+        // checks to see if captions already exist in the database
+        if (document.getElementById("FullCap").innerHTML == "") {
+            fetch('/fullcaption?id=' + current_vID, {
+                method: 'GET',
+            }).then((response) => response.text()).then((text) => {
+                if (text.length > 0) {
+                    // Sets the results table
+                    document.getElementById("FullCap").innerHTML = text;
+                    console.log("Fetching full captions from database...");
```

Comment:
I believe the full caption has been fetched at this point right?

Reply:
yeah, log was just for me i had just copied it from another log message, will take it out.